dat/missions/neutral/pirbounty_dead.lua

```diff
@@ -295,6 +306,7 @@ function pilot_death( p, attacker )
          hook.timer( 3000, "timer_hail", top_hunter )
          misn.osdDestroy()
       else
+         misn.appendLog("Pirate killed, but not by you")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.